foundry_ml.egg-info/requires.txt

```diff
@@ -0,0 +1,10 @@
+mdf_forge>=0.8.0
```

Comment:
I see what's going on here -- please remove the entire `foundry_ml.egg-info` folder :)